predictor_robojudge.ipynb

```diff
@@ -427,34 +439,6 @@
     "        print(errors)"
    ]
   },
```

Comment:
Use new function instead of these individual cells

Reply:
👍